docs/_templates/layout.html

```diff
@@ -6,7 +6,12 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>{{ title|striptags|e }}{{ titlesuffix }}</title>
-  {%- block extrahead %} {% endblock %}
+  <!-- begin extra head -->
+  {%- block extrahead %}
+    {% if pagename.endswith("api") %}
+      <script type="text/javascript" src="./_static/api_redirect.js"></script>
+    {% endif %}
```

Comment:
i think it's better to keep it inside `<head>` since this script modifies `document.location` to redirect, and we want to redirect as fast as possible and not waste time trying to render page which users still wont see

Reply:
Definitely, but the other one is also still in the `<head>`; all scripts should go into the `scripts` template block to keep things organized :p This being in the `extrahead` block also means a child template could overwrite it.

Reply:
fair